consensus/bor/validator.go

```diff
@@ -80,18 +82,6 @@ func ValidatorListString(vals []*Validator) string {
 	return strings.Join(chunks, ",")
 }
 
-// Bytes computes the unique encoding of a validator with a given voting power.
-// These are the bytes that gets hashed in consensus. It excludes address
-// as its redundant with the pubkey. This also excludes ProposerPriority
-// which changes every round.
-func (v *Validator) Bytes() []byte {
-	b, err := json.Marshal(v)
-	if err != nil {
-		return b
-	}
-	return nil
-}
-
```

Comment:
Why this method is removed? This can be used in encoding/decoding while the validator is getting stored in the store.